package.json

```diff
@@ -53,18 +53,16 @@
     "webpack": "^3.5.5"
   },
   "scripts": {
-    "clean": "rm -fr node_modules test/node_modules",
+    "clean": "rm -fr node_modules",
     "clean:public": "bin/cleanPublic.js",
     "compile": "bin/compile.js",
     "pregenerate": "yarn compile",
     "generate": "bin/generate.js",
     "preghpages": "yarn generate",
     "ghpages": "bin/ghpages.js",
     "lint": "eslint -c .eslintrc.json src",
-    "postinstall": "cd test; yarn install; yarn run webdriver:update; cd -;",
```

Comment:
Had to remove this, because it would run (and throw an error) via `npm install helix-ui` in a consuming application.

Reply:
I always forget this. Apologies!

Also, I don't like that behavior in `npm`/`yarn`. I wish there was a way to specify that an install is happening as a developer versus a consumer, like a `pre`/`post` `install`, there'd be a `pre`/`post` `install-dev` task as well.